src/watcher.js

```diff
@@ -384,7 +384,7 @@ class Watcher extends SafeEventEmitter {
 
             channel = currentChannel;
 
-            api.get(`channels/${channel.name}`)
+            api.get(`users/twitch/${channel.id}`)
                 .catch(error => ({
                     bots: [],
                     emotes: [],
```

Comment:
if an error occurs you will cause an error since this fake response data is not identical in format to the normal real response